lib/busbar_cli/commands/logs.rb

```diff
@@ -3,43 +3,34 @@ module Logs
     extend ActiveSupport::Concern
 
     included do
-      desc 'logs APPLICATION_CONTAINER ENV [COMPONENT_TYPE]',
-        'Fetch the logs from a application container. You can find the container for your application through the "busbar containers" command.'
+      desc 'logs APP ENV [COMPONENT]',
+        'Fetch the logs from a application.'
       method_option :since,
                     default: '0',
                     type: :string,
                     desc: 'Only return logs newer than a relative duration like 5s, 2m, or 3h. ' \
```

Comment:
What do you think of changing these two lines to:

```suggestion
                     desc: 'Look-back duration with units like 5s, 2m, or 3h.'
```

Note that the next line (`'Defaults to all logs.'`) should be deleted because now it's clear what `Default: 0` means. See previous comment regarding where `Default: 0` comes from.

Also, I find it ambiguous if `2m` means two _minutes_ or two _months_. Do you know which library we're using for this translation? I think we should put a reasonable maximum cap like `h` (hours) or `d` (days) so that users don't do unexpected things like trying to stream a year's worth of logs.

Reply:
That last space is needed because it concatenates that line with the one bellow it.